easybuild/easyconfigs/n/NGS/NGS-2.10.9-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'Bundle'
```

Comment:
Missing an author attribution from previous versions here.

Reply:
This easyconfig is based on `NGS-2.10.4-GCCcore-8.3.0-Java-11.eb` (which I authored) so it's ok as it is :)